lib/presentation/mixins/single_image_picker_mixin.dart

```diff
@@ -23,8 +23,9 @@ mixin SingleImagePickerMixin on CommonMediaPickerMixin {
   Future<void> showImagePickerBottomSheet(
     BuildContext context,
     PermissionStatus? permissionStatusPhotos,
-    ImagePickerGridController imagePickerController,
-  ) async {
+    ImagePickerGridController imagePickerController, {
+    RequestType type = RequestType.common,
```

Comment:
because it is the `single_image_picker_mixin` please use `RequestType.image` is default value. Thanks